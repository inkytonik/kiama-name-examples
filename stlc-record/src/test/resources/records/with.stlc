with {a=1} do a+1
