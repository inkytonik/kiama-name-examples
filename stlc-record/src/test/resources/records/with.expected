src/test/resources/records/with.stlc:1:1:info: with {a =  ...: num
with {a=1} do a+1
^
src/test/resources/records/with.stlc:1:6:info: {a = 1}: {a : num}
with {a=1} do a+1
     ^
src/test/resources/records/with.stlc:1:7:info: a def: field in value num
with {a=1} do a+1
      ^
src/test/resources/records/with.stlc:1:9:info: 1: num
with {a=1} do a+1
        ^
src/test/resources/records/with.stlc:1:15:info: a + 1: num
with {a=1} do a+1
              ^
src/test/resources/records/with.stlc:1:15:info: a: num
with {a=1} do a+1
              ^
src/test/resources/records/with.stlc:1:17:info: 1: num
with {a=1} do a+1
                ^
num
